app-mobilephone/gammu/gammu-1.38.4.ebuild

```diff
@@ -3,32 +3,33 @@
 
 EAPI=6
 
-inherit eutils cmake-utils
+inherit cmake-utils eutils
 
 DESCRIPTION="A tool to handle your cellular phone"
 HOMEPAGE="https://wammu.eu/gammu/"
-SRC_URI="https://dl.cihar.com/${PN}/releases/${P}.tar.bz2"
```

Comment:
Please restore the original order of variables; it's standard throughout different ebuilds and makes it easier to read (= harder to miss something). Only data inside the vars is sorted.